src/app/common/errors.controller.js

```diff
@@ -46,6 +50,12 @@
         return gettextCatalog.getString('There is an error in your registration. You may have already registered. If so, simply <a href="/password_reset" target="_top">reset your password</a>.');
       }
 
+      // List not found — the template shows a good error message so we will
+      // suppress the modal from displaying.
+      if (rejection.config.method === 'GET' && rejection.config.url.indexOf('/list/')) {
```

Comment:
@rupl I'm thinking maybe it would be better to use HTTP status codes here, and display a modal only when status !== 404 && status !== 400 ?

Reply:
Sure I can explicitly check codes. However, the ticket was created specifically to deal with a 500 coming back. That's why I want to be careful and not under-report 500s that maybe do need to be shown to the user (in cases where a template isn't providing context).

Reply:
Come to think of it, we do want to suppress the modal when it's 404. The page looks like this already — spacing fixed in #175

![image](https://user-images.githubusercontent.com/254753/66482651-b3e1cf80-eaa3-11e9-9175-11178a3ca05a.png)